HtmlEditor: image upload api (#3647)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxHtmlEditor/1 Configuration/imageUpload/imageUpload.md

---
id: dxHtmlEditor.Options.imageUpload
type: dxHtmlEditorImageUpload
inheritsType: dxHtmlEditorImageUpload
default: { tabs: ['url'], fileUploadMode: 'base64', uploadUrl: undefined, uploadDirectory: undefined }
---
---
##### shortDescription
Configures the image upload.

---
#include common-demobutton with {
    url: "https://js.devexpress.com/Demos/WidgetsGallery/Demo/HtmlEditor/Overview/"
}

Click the 'Add Image' [toolbar button](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/toolbar/items/) to invoke the 'Add an Image' dialog.

![DevExtreme Html Editor - Upload Images Dialog](/images/htmlEditor/image-upload-dialog.png)

> Use the [fileUploadMode](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/imageUpload/#fileUploadMode) property to specify whether to upload images on the server or to the Base64 binary format. The [tabs](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/imageUpload/#tabs) property specifies the visibility of tabs in the 'Add Image' dialog. 

Reviewer: > Use the [fileUploadMode](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/imageUpload/#fileUploadMode) property to specify whether **to upload images on the server or to the Base64 binary format**.

The Base64 binary format is just a data format, not a destination for the image upload process, unlike the server. Therefore, the sentence is not clear. I believe you meant something like this:

```suggestion
Use the [fileUploadMode](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/imageUpload/#fileUploadMode) property to specify whether to upload images as is or in Base64 binary format. The [tabs](/Documentation/ApiReference/UI_Components/dxHtmlEditor/Configuration/imageUpload/#tabs) property specifies the visibility of tabs in the 'Add Image' dialog. 
```

Since we use the format name as circumstances -- in format, that means we need no article here.
